### ISSUE TITLE
Adding a new question does not work with Django 5

### ISSUE THREAD

**Issue by frlan** [smark-1/django-easy-faq#15]
With Django 5.0 I'm getting this traceback:

```
Traceback (most recent call last):
  File "/my/project/.venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 714, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/utils/decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/utils/decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1941, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/utils/decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/utils/decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1802, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1853, in _changeform_view
    self.save_model(request, new_object, form, not add)
  File "/my/project/.venv/lib/python3.11/site-packages/django/contrib/admin/options.py", line 1257, in save_model
    obj.save()
  File "/my/project/.venv/lib/python3.11/site-packages/faq/models.py", line 28, in save
    self.helpful = self.get_helpful()
                   ^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/faq/models.py", line 18, in get_helpful
    return QuestionHelpful.objects.filter(question=self, vote=True).count()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1600, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1632, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1546, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1376, in build_lookup
    lookup = lookup_class(lhs, rhs)
             ^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/lookups.py", line 30, in __init__
    self.rhs = self.get_prep_lookup()
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/fields/related_lookups.py", line 147, in get_prep_lookup
    self.rhs = get_normalized_value(self.rhs, self.lhs)[0]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/my/project/.venv/lib/python3.11/site-packages/django/db/models/fields/related_lookups.py", line 45, in get_normalized_value
    raise ValueError("Model instances passed to related filters must be saved.")
ValueError: Model instances passed to related filters must be saved.
```
Running 
```
Django==5.0
django-easy-faq==1.6
```
on Python 3.11 and Debian.

**Comment by smark-1** [comment]
This was also an issue with new answers

**Comment by smark-1** [comment]
This issue is now fixed with the latest release 1.7

**Comment by frlan** [comment]
This was fast :+1:
